locales/es/messages.po

```diff
@@ -2780,7 +2780,7 @@ msgid "justfix.commonWarningAboutChangingLandlordDetails"
 msgstr "Si recibes documentos relacionados con el alquiler desde una dirección diferente, o si el dueño de tu edificio te ha dado instrucciones de usar una dirección diferente para vuestra correspondencia, puedes <0>proporcionar tus propios datos.</0> Sin embargo, sólo debes usar una dirección distinta a la que te sugerimos si tienes la certeza absoluta de que es correcta — es más seguro usar la dirección oficial que el dueño de tu edificio ha registrado con la ciudad."
 
 #: frontend/lib/data-driven-onboarding/data-driven-onboarding.tsx:337
-msgid "justfix.ddoEfnycCovidMessage"
+msgid "justfix.ddoEvictionfreeCovidMessage"
```

Comment:
same q here

Reply:
Cool, same answer as above!

Oh also, a thing to note about the localization message catalogs (the files ending in `.po`) is that the non-English ones are actually _overwritten_ by Crowdin, our localization platform.  The platform offers a way to manually upload a non-English `.po` file to their site to "cancel out" the fact that its translations will be out-of-sync with the changes you're making here... So you might want to do that.

Alternatively, you could just keep the original ID here, since the whole thing is a bit of a hassle and it's not that big a deal to have a localization ID with a kind of wonky name in it.  Up to you!

Reply:
In the spirit of hoping we can retire this Covid message sometime soon anyways, I think I'll leave it as is. :)